src/libraries/System.Text.Json/tests/System.Text.Json.SourceGeneration.Tests/TestClasses.cs

```diff
@@ -158,4 +158,23 @@ public struct PersonStruct
         public string FirstName { get; set; }
         public string LastName { get; set; }
     }
+
+    public class TypeWithValidationAttributes
+    {
+        [ComponentModel.DataAnnotations.Required(ErrorMessage = "Name is required")]
```

Comment:
Can you add another property where with a `[Required]` attribute but no properties?

Reply:
OK, done.